where-to-travel/src/main/webapp/script.js

```diff
@@ -808,19 +814,28 @@ function getLoginHtml() {
 }
 
 /**
- * A helper function that returns the HTML for the user dashboard given a user.
+ * A helper function that returns the HTML for the user dashboard given a user and
+ * adds name to text on top bar.
  * 
- * @param {User} user
  * @returns the HTML for user dashboard as a string 
  */
-function getUserDashHtml(user) {
-  return `<img onclick="showModal(${INFO_HTML_PATH})" class="btn btn-icon" src="icons/help.svg">
-          Display Saved:
-          <label class="switch btn">
-            <input type="checkbox">
-            <span class="slider round"></span>
-          </label>
-          <a class="btn btn-outline-primary btn-color" style="color: #049688;" id="logout">Logout</a>`;
+function getUserDashHtml() {
+  return new Promise(function(resolve) { 
+     firebase.database().ref('users/'+ firebase.auth().currentUser.uid)
+      .once('value', function(userSnapshot){
+        const name = userSnapshot.val().name;
+        document.getElementById('top-text').innerHTML =
```

Comment:
Do we want to create a constant for 'top-text'?

Reply:
Fixed!